webui/react/src/pages/InteractiveTask.tsx

```diff
@@ -9,7 +9,7 @@ import { CommandState, CommandType } from 'types';
 import css from './InteractiveTask.module.scss';
 import TaskLogs from './TaskLogs';
 
-interface Params {
+type Params = {
```

Comment:
**question:** related to the question written below, should `taskId`, `taskName` and `taskResourcePool` be written with the `?` suffix (e.g. `taskId?: string;`)?

much like `ProjectDetails.tsx Params`

Reply:
in `react-router-dom v6`, `useParam` returns always `string | undefined` even when we don't put `?`. 
In v5, `useParam` returns only string if we dont put `?`, so it used be like [this](https://github.com/determined-ai/determined/blob/2da2ed2a1f11a87bd5999200561671149bd10cc9/webui/react/src/pages/InteractiveTask.tsx#L12). I just followed the previous style which doesnt have `?`. 
Also, I replaced `interface` with `type` due to type error in a new version.